modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -407,8 +408,8 @@ include::devguide:example$nodejs/capella_connection_crud.ts[tag=replace,indent=0
 
 ====
 
-CAUTION: When you replace a document, it's usually good practice to use xref:howtos:concurrent-document-mutations.adoc[optimistic locking].
-Otherwise, changes might get lost if two people change the same document at the same time.
+CAUTION: Always practice to use xref:howtos:concurrent-document-mutations.adoc[optimistic locking] when you replace a document. 
+Else, changes might get lost if 2 people change the same document at the same time.
```

Comment:
Is this an American English thing? Otherwise feels more natural here.
I don't think it's in the style guide either way.